test/test_ops.py

```diff
@@ -1577,6 +1577,121 @@ def is_bit_set(x):
             torch.is_complex,
         )
 
+    @onlyCUDA
+    def test_opinfo_aten_ops_coverage(self):
+        all_aten_ops = {
+            torch.ops.aten.__getattr__(attr)
+            for attr in dir(torch.ops.aten)
+            if not attr.startswith('__') and attr not in {'name'} and
+            isinstance(torch.ops.aten.__getattr__(attr), torch._ops.OpOverloadPacket)
+        }
+
+        # Copies inputs to inplace operations to avoid inplace modifications
+        #   to leaves requiring gradient
+        def _get_safe_inplace(inplace_variant):
+            @wraps(inplace_variant)
+            def _fn(t, *args, **kwargs):
+                return inplace_variant(t.clone(), *args, **kwargs)
+            return _fn
+
+        seen_aten_ops = set()
+
+        class CollectDispatchMode(torch.utils._python_dispatch.TorchDispatchMode):
+            def __torch_dispatch__(self, func, types, args=(), kwargs=None):
+                seen_aten_ops.add(func.overloadpacket)
+                kwargs = kwargs or {}
+                return func(*args, **kwargs)
+
+        for device in ["cpu", "cuda"]:
+            for op in op_db:
+                supported_dtypes = op.supported_backward_dtypes(device) or op.supported_dtypes(device)
+
+                if not supported_dtypes:
+                    continue
+
+                dtype = next(iter(supported_dtypes))
+
+                func = op.get_op()
+                inplace_func = op.get_inplace()
+                if inplace_func:
+                    inplace_func = _get_safe_inplace(inplace_func)
+
+                requires_grad = (
+                    op.supports_autograd
+                    and dtype in op.supported_backward_dtypes(torch.device(device).type)
+                    # TODO: OpInfo really ought to error out for this case, but it's
+                    # not exercised in test_ops_gradients atm.  The problem is not
+                    # complex32 per-se (which is supported by data movement only ops)
+                    # but that when we do backwards we expect other ops like add to work
+                    and not dtype == torch.complex32
+                )
+
+                samples = op.sample_inputs(device, dtype, requires_grad=requires_grad)
+                for sample_input in samples:
+                    args = [sample_input.input] + list(sample_input.args)
+                    kwargs = sample_input.kwargs
+
+                    try:
+                        with CollectDispatchMode():
```

Comment:
I'm confused about this part. The operation is called and the backward pass is triggered, but exceptions are ignored?